README.md

```diff
@@ -10,15 +10,17 @@ Please note: Issues on this repository are intended to be related to bugs or fea
 
 ## Using the Provider
 
-This Terraform Provider is not available to install automatically via `terraform init` at this time. Instead, follow the instructions to [install it as a plugin](https://www.terraform.io/docs/plugins/basics.html#installing-plugins). Pre-built releases of this Terraform Provider are available for download in [GitHub Releases](https://github.com/bflad/terraform-provider-time/releases). After placing the custom provider into your plugins directory, run `terraform init` to initialize it.
-
-### Resource Documentation
-
-Until this Terraform Provider is brought under the Terraform Provider development program, resource documentation can be found within this repository.
+This Terraform Provider is available to install automatically via `terraform init`. It is recommended to setup the following Terraform configuration to pin the major version:
+
+```hcl
+terraform {
+  required_providers = {
+    time = "~> 0.3"
```

Comment:
If automation isn't bumping this, I'd just maybe put like a `x.x` or something here with a comment to fill it in.

Reply:
Updating via https://github.com/hashicorp/terraform-provider-time/pull/5/commits/4e035c74fa0c2e1065b99d5e6ff6bcb92dd51d29